packages/page-accounts/src/Sidebar/Balances.tsx

```diff
@@ -21,17 +21,12 @@ function Balances ({ address, className }: Props): React.ReactElement<Props> | n
   return (
     <section className={className}>
       <div className='ui--AddressMenu-sectionHeader'>
-        <div>
-          <Icon icon='sort-amount-down' />
-          &nbsp;
-          {t<string>('balance')}
-        </div>
+        {t<string>('balance')}
       </div>
       <AddressInfo
         address={address}
         className='balanceExpander'
         withBalance={WITH_BALANCE}
-        withBalanceToggle
```

Comment:
Not sure those changes go with this PR. Seems unrelated to Account row changes.

Reply:
@MiZiet - does your styling change require resigning from toggle'able address info in other places?

Reply:
I can revert changes in sidebar. I've styled balances in details row and in sidebar. I agree with @krzysztof-jelski that sidebar changes shouldn't be in this pr.

Reply:
✅